aspnetcore/fundamentals/static-files.md

```diff
@@ -1,17 +1,1150 @@
 ---
 title: Static files in ASP.NET Core
 author: wadepickett
-description: Learn how to serve and secure static files and configure static file hosting middleware behaviors in an ASP.NET Core web app.
+description: Learn how to serve and secure static files and configure Map Static Assets endpoint conventions and Static File Middleware in ASP.NET Core web apps.
 monikerRange: '>= aspnetcore-3.1'
 ms.author: wpickett
 ms.custom: mvc
-ms.date: 03/18/2025
+ms.date: 09/02/2025
 uid: fundamentals/static-files
 ---
 # Static files in ASP.NET Core
 
 [!INCLUDE[](~/includes/not-latest-version.md)]
 
-[!INCLUDE[](~/fundamentals/static-files/includes/static-files8.md)]
+Static files, also called static assets, are files an ASP.NET Core app that aren't dynamically generated and served directly to clients on request, such as HTML, CSS, image, and JavaScript files.
 
-[!INCLUDE[](~/fundamentals/static-files/includes/static-files6.md)]
+For Blazor static files guidance, which adds to or supersedes the guidance in this article, see <xref:blazor/fundamentals/static-files>.
+
+:::moniker range=">= aspnetcore-9.0"
+
+To enable static file handling in ASP.NET Core, call <xref:Microsoft.AspNetCore.Builder.StaticAssetsEndpointRouteBuilderExtensions.MapStaticAssets%2A>. 
+
+By default, static files are stored within the project's [web root](xref:fundamentals/index#web-root) directory. The default directory is `{CONTENT ROOT}/wwwroot`, where the `{CONTENT ROOT}` placeholder is the app's [content root](xref:fundamentals/index#content-root). Only files in the wwwroot folder will be addressable, so you don't need to worry about the rest of your code.
+
+Only files with specific file extensions mapped to supported media types are treated as static web assets.
```

Comment:
@javiercn Is there more we should say about the default media type mapping for static web assets and how it can be configured?

Reply:
Yes, I included a section about that in my original technical description. it should be covered here if it's not covered later in the document. At leas a note/link here to where it can be configured later.